schedule/yast/select_disk.yaml

```diff
@@ -0,0 +1,42 @@
+---
+description: >
+  Test the selection of "first" disk with the guided setup in partitioning.
+  This is also used as a prerequisite for real hardware tests to select
+  the right disk for installation and not a "random" one.
+name: select_disk
+vars:
+  SELECT_FIRST_DISK: 1
```

Comment:
From `git grep` I can see that `SELECT_FIRST_DISK` is only used for scheduling, so we could skip it here. Could you please create follow up ticket to migrate this scenario for openSUSE too, so we can potentially get rid of this variable?

Reply:
https://progress.opensuse.org/issues/68720